src/components/LandingPage.jsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 
 const LandingPage = () => {
+  const [user, setUser] = useState([]);
+
+  useEffect(() => {
+    fetch('/home')
+      .then(res => res.json())
```

Comment:
One other advantage of axios over fetch is the direct conversion of the response to `json` internally, so this step won't be needed.